pkg/nvcdi/driver-nvml.go

```diff
@@ -220,3 +230,37 @@ func getVersionLibs(logger logger.Interface, driver *root.Driver, version string
 
 	return relative, nil
 }
+
+func (d driverVersionDiscoverer) Hooks() ([]discover.Hook, error) {
+	mounts, err := d.Discover.Mounts()
+	if err != nil {
+		return nil, fmt.Errorf("failed to get library mounts: %v", err)
+	}
+
+	var links []string
+	for _, mount := range mounts {
+		dir, filename := filepath.Split(mount.Path)
+		// TODO: We should include the other libraries as is done here:
+		// https://github.com/NVIDIA/nvidia-container-toolkit/blob/79c59aeb7f59dd612793ac80a8d7022c554634bb/internal/platform-support/tegra/symlinks.go#L84-L97
```

Comment:
The code you linked for tegra creates `ibcuda.so -> libcuda.so.1` while the code below seems to be creating `ibcuda.so -> libcuda.so.RM_VERSION`. Why the difference?

Reply:
You're right. I was trying to simplify things by linking to the `.RM_VERSION`, but since we have the following the driver manifest:
```
libcuda.so.550.120 0755 CUDA_LIB NATIVE / MODULE:gpgpu
libcuda.so.1 0000 CUDA_SYMLINK NATIVE / libcuda.so.550.120 MODULE:gpgpu
libcuda.so 0000 CUDA_SYMLINK NATIVE / libcuda.so.1 MODULE:gpgpu
```
we should link to `libcuda.so.1`.

Reply:
Yes, the smylink should be to `libcuda.so.1` not the actual `.RM_VERSION` file.

Reply:
Updated in latest.